R/jss_article.R

```diff
@@ -5,12 +5,17 @@
 #' @inheritParams rmarkdown::pdf_document
 #' @param ... Arguments to \code{rmarkdown::pdf_document}
 #' @export
-jss_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
+jss_article <- function(..., keep_tex = TRUE, citation_package = 'natbib',
+                        pandoc_args = NULL) {
```

Comment:
When possible, I write all arguments on one line; if not possible, I use this style:

```suggestion
jss_article <- function(
  ..., keep_tex = TRUE, citation_package = 'natbib',
  pandoc_args = NULL
) {
```

Reply:
I can adapt to this style already in place in **rticles**. 

I used that because RStudio IDE aligns function arguments this way when I go to next line so it is practical ☺️ 
And it is coherent with https://style.tidyverse.org/ that I know of.